### PR TITLE
Jv spring rest

### PR DESCRIPTION
**Pull request by VolodymyrKachan**

### CODE REVIEW

File: src/main/java/mate/academy/spring/controller/MovieSessionController.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/movie_session")
```
Comment (GolDenik) [inline src/main/java/mate/academy/spring/controller/MovieSessionController.java:23]:
```suggestion
@RequestMapping("/movie-sessions")
```

File: src/main/java/mate/academy/spring/controller/MovieSessionController.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+                .add(movieSessionMapper.toModel(movieSessionRequestDto)));
+    }
+
+    @PutMapping("/{id}")
+    public MovieSessionResponseDto update(
+            @PathVariable Long id, @RequestBody MovieSessionRequestDto movieSessionRequestDto) {
+        MovieSession movieSession = movieSessionMapper.toModel(movieSessionRequestDto);
+        movieSession.setId(id);
+        MovieSession movieSessionUpdate = movieSessionService.update(movieSession);
+        return movieSessionMapper.toDto(movieSessionUpdate);
```
Comment (GolDenik) [inline src/main/java/mate/academy/spring/controller/MovieSessionController.java:56]:
```suggestion
        return movieSessionMapper.toDto(movieSessionService.update(movieSession));
```

**Review by GolDenik — APPROVED** [review]

File: src/main/java/mate/academy/spring/controller/CinemaHallController.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+import mate.academy.spring.mappers.CinemaHallMapper;
+import mate.academy.spring.service.CinemaHallService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/cinema_halls")
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/controller/CinemaHallController.java:16]:
```suggestion
@RequestMapping("/cinema-halls")
```
**check all endpoints**

File: src/main/java/mate/academy/spring/controller/MovieSessionController.java [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        MovieSession movieSession = movieSessionMapper.toModel(movieSessionRequestDto);
+        movieSession.setId(id);
+        return movieSessionMapper.toDto(movieSessionService.update(movieSession));
+    }
+
+    @DeleteMapping("/{id}")
+    public void delete(@PathVariable Long id) {
+        movieSessionService.delete(movieSessionService.get(id));
+    }
+
```
Comment (Sofasmile) [inline src/main/java/mate/academy/spring/controller/MovieSessionController.java:62]: remove empty line

**Review by Sofasmile — CHANGES_REQUESTED** [review]

**Review by Sofasmile — APPROVED** [review]

**Pull request closed without merging**